src/index.js

```diff
@@ -189,4 +186,4 @@ function getProxyTargetForValue (value) {
   }
 }
 
-module.exports = { Membrane }
```

Comment:
remove this change

Reply:
No! We must export object graph!

Reply:
actually considering the lavamoat security assumptions you are correct. we should export all classes on module.exports